pkg/util/cmpver/cmpver.go

```diff
@@ -130,8 +131,13 @@ func validateOperation(op Operation) error {
 }
 
 func isLatest(version string) bool {
-	switch version {
-	case "latest", "nightly":
+	switch {
```

Comment:
please update above comments.

> // Latest or nightly version is larger than any version

Reply:
Updated in 793aaa46222d8d980524d9e95a834c6f583e87f0